tests/Integration/CapabilityTest.php

```diff
@@ -17,7 +17,7 @@ final class CapabilityTest extends TestCase {
 	 */
 	public function tearDown() {
 
-		(new Capability())->unregister();
+		( new Capability() )->unregister();
```

Comment:
```suggestion
		(new Capability())->unregister();
```

These code style changes should be in a different PR.